_includes/install_command.html

```diff
@@ -0,0 +1,28 @@
+{%- capture code -%}
+brew {{include.modifier}} install {{include.item}}
+{%- endcapture -%}
+
+<style>
+    #copyButton {
```

Comment:
Styling wise this looks alright but might be nice to have a background like the installation command on https://brew.sh and some label that explains what this text is.

Reply:
![image](https://user-images.githubusercontent.com/10703445/94144127-61edb180-fe8e-11ea-85d4-a3f3c1c120d9.png)

Does this look good?

Reply:
Yes, that's looking much better! How about putting the 📋 inside the background area?

Reply:
![image](https://user-images.githubusercontent.com/10703445/94145076-9f067380-fe8f-11ea-99bf-dd17a7390e9c.png)

Reply:
![image](https://user-images.githubusercontent.com/10703445/94147470-f0643200-fe92-11ea-9b35-444c2dd59dbe.png)
![image](https://user-images.githubusercontent.com/10703445/94147600-1db0e000-fe93-11ea-8b62-c1e722da1586.png)

I had tried that first, but it's displayed weird. 

My CSS is a bit rusty any pointers?

Reply:
![image](https://user-images.githubusercontent.com/10703445/94148498-54d3c100-fe94-11ea-8763-09b238254140.png)

Which component is being used here?

Reply:
> I had tried that first, but it's displayed weird.
> 
> My CSS is a bit rusty any pointers?

I could try and help once it's committed here. Not sure off the top of my head.



> Which component is being used here?

All the HTML/CSS for that is in https://github.com/homebrew/brew.sh. Hopefully that helps. It would be nice to match that, I agree. We might want to move more CSS into https://github.com/homebrew/brew.sh (as it's shared on this site)

Reply:
I've used the syntax highlights same as brew.sh now. 

Can you share which files I should modify to speed up the start time?

It's taking upwards of 30minutes here for every build 😞 .

Reply:
![image](https://user-images.githubusercontent.com/10703445/94156937-44284880-fe9e-11ea-9175-e675ac98dd0a.png)

This looks nice too, maybe we can add a border similar to that around the button?

![image](https://user-images.githubusercontent.com/10703445/94157406-d6c8e780-fe9e-11ea-95f6-93b5c8c581c7.png)

But that's all adding more code, or should we just keep it simple?

Reply:
![image](https://user-images.githubusercontent.com/10703445/94158448-062c2400-fea0-11ea-8898-45bee178adda.png)

Adding a line before the icon should make it okay right?

Reply:
![image](https://user-images.githubusercontent.com/10703445/94158661-4d1a1980-fea0-11ea-896a-b28fa1e7dd57.png)

Reply:
> I've used the syntax highlights same as brew.sh now.

Nice!




> Can you share which files I should modify to speed up the start time?

If you rebase on `origin/master` you should find `rake build` and `rake serve` should run a bit quicker and without depending on each other.



> Adding a line before the icon should make it okay right?

I think it's probably nicer without the border?

Reply:
A separation is good for user context too right? Otherwise they might just copy the entire line including the paste unicode and paste it in the shell.